lib/matplotlib/cbook.py

```diff
@@ -2445,6 +2445,14 @@ def safe_first_element(obj):
     return next(iter(obj))
 
 
+def asSequence(data):
```

Comment:
Naming is hard.  A problem here is that where this is used, there is no clue that the "view" in question is the obscure "MappingView", not a "view" in the numpy sense, which is much more common. "convert_dict_view"?  "convert_MappingView"?